OpenRA.Game/Widgets/WorldInteractionControllerWidget.cs

```diff
@@ -120,12 +120,14 @@ public override bool HandleMouseInput(MouseInput mi)
 							}
 						}
 					}
-					else if (dragStart.HasValue)
+					else if (dragStart.HasValue && (!(World.OrderGenerator is GenericSelectTarget) || hasBox))
```

Comment:
Whoops, I just noticed StyleCop complains about the comment's location. The way I did it now is the only way it wouldn't complain about the comment. Should we disable that rule (SA1108), to make comments like below possible?

```csharp
if (x)
{
    foo();
}
// Comment
else if (y)
{
    bar();
}